### ISSUE TITLE
Some routes the package simply don't minify

### ISSUE THREAD

**Issue by brucedeity** [fahlisaputra/laravel-minify#29]
For example, route "../user/1" it minifies, but "../user/2" it simply doesnt minify the output html, and no error log is available at browser console or anything, so it's very unclear what is the actual reason.

**Comment by mcolominas** [comment]
Does the route "../user/2" exist? If it doesn't exist and you get a 404 error code, that's the problem, the web middleware only works for 200 code requests, if you're using Laravel 11 try using:

```php
$middleware->append([
    \Fahlisaputra\Minify\Middleware\MinifyHtml::class,
    \Fahlisaputra\Minify\Middleware\MinifyCss::class,
    \Fahlisaputra\Minify\Middleware\MinifyJavascript::class,
]);
```

It works for me, without more information about what your request returns, there is little that can be done.

**Comment by brucedeity** [comment]
Yes, both routes exists.. but i simply moved to another laravel minify project
